### ISSUE TITLE
Terminal window pops up when trying to Load Paper Tape

### ISSUE THREAD

**Issue by kidmirage** [obsolescence/pidp1#6]
The terminal window has this WARNING message:

> (zenity:2973): Gtk-WARNING **: 12:01:20.461: Unable to acquire the address of the accessibility bus: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: The name org.a11y.Bus was not provided by any .service files. If you are attempting to run GTK without a11y support, GTK_A11Y should be set to 'none'.

The select dialog for the tape works fine and the tape loads successfully. When the dialog closes the terminal window also disappears. So this is not a critical issue.
